homework-submission/homework-week1/App.js

```diff
@@ -0,0 +1,84 @@
+class ListItem extends React.Component {
+  // ListItem component is creating a list item in page
+  render() {
+    const { description, deadline } = this.props;
+
+    return (
+      <li key={this.props.id} className={this.props.done ? 'itemCompleted' : 'itemNotCompleted'}>
+        {description} , {deadline}
+      </li>
+    );
+  }
+}
+
+class StaticList extends React.Component {
+  // StaticList component is creating a list by using the ListItem component in page statically
+  render() {
+    return (
+      <ul>
+        <ListItem id="1" description="Get out of bed" deadline="Wed Sep 13 2017" />
+        <ListItem id="2" description="Brush teeth" deadline="Thu Sep 14 2017" />
+        <ListItem id="3" description="Eat breakfast" deadline="Fri Sep 15 2017" />
+      </ul>
+    );
+  }
+}
+
+class DynamicList extends React.Component {
+  // DynamicList component is creating a list by using the ListItem component in page dynamically
+  render() {
+    const { todoList } = this.props;
+    return (
+      <ul>
+        {todoList.map(item => {
+          return (
+            <ListItem
+              key={item.id * 3}
```

Comment:
Why do you multiply the id with 3? :)

Reply:
Actually that is meaningless, It would be key={item.id}